evm_arithmetization/src/cpu/kernel/constants/journal_entry.rs

```diff
@@ -11,10 +11,11 @@ pub(crate) enum JournalEntry {
     Refund = 8,
     AccountCreated = 9,
     Log = 10,
+    TransientStorageChange = 11,
```

Comment:
You don't need to state the discriminant I think. The only time you need to do that is when you want non-ordered discriminants, e.g. `enum Thing{A = 3, B = 1, C = 4}`.

Reply:
I'd say for clarity either we remove all of them, or we keep them specified, otherwise it may look confusing.

Reply:
I'm suggesting we remove all of them.